app/models/language/tag.rb

```diff
@@ -0,0 +1,179 @@
+# frozen_string_literal: true
+
+class Language::Tag < ApplicationRecord
+  has_many :tag_subtags
```

Comment:
I did a search through the PR changes and I only see a reference to `tag_subtag` here, in a migration, and in the schema.  Is it used elsewhere?

Reply:
ah, I think this is a relic of `has_many through:` in an earlier iteration

Reply:
the migration supports the HABTM

Reply:
@barmintor and I talked this over and we think that `has_many :tag_subtags` can be deleted (assuming tests still pass).

Reply:
This association was extraneous and deleted.